modules/stereo/samples/Sample1.cpp

```diff
@@ -0,0 +1,78 @@
+#include <iostream>
+#include "opencv2/stereo.hpp"
+#include "opencv2/imgproc.hpp"
+#include "opencv2/highgui.hpp"
+#include <iostream>
+
+using namespace cv;
+using namespace stereo;
+using namespace std;
+
+//in this example we will load a sequence of images from a file process them and display the result on the screen
+//the descriptor used is the modified_census transform
+
+int main(int, char**)
+{
+    //begin the program
+    cout << " Running Main function \n";
+    //declare 2 images
+    Mat image1, image2;
+
+    //  -- 1. Call the constructor for StereoBinaryBM
+    int ndisparities = 32;   /**< Range of disparity */
+    int kernelSize = 9; /**< Size of the block window. Must be odd */
+
+    Ptr<StereoBinaryBM> sbm = StereoBinaryBM::create(ndisparities, kernelSize);
+
+    // -- 2. Set parameters
+    sbm->setPreFilterCap(31);
+    sbm->setMinDisparity(0);
+    sbm->setTextureThreshold(10);
+    sbm->setUniquenessRatio(0);
+    sbm->setSpeckleWindowSize(400);//speckle size
+    sbm->setSpeckleRange(200);
+    sbm->setDisp12MaxDiff(0);
+    sbm->setScalleFactor(4);//the scalling factor
+    sbm->setBinaryKernelType(CV_MODIFIED_CENSUS_TRANSFORM);//binary descriptor kernel
+    sbm->setAgregationWindowSize(9);
+    sbm->setSpekleRemovalTechnique(CV_SPECKLE_REMOVAL_AVG_ALGORITHM);//speckle removal algorithm
+    sbm->setUsePrefilter(false);//prefilter or not the images prior to making the transformations
+
+    for(int i = 0 ; i < 200; i++)
+    {
+        string path = "D:\\WorkingSec";
```

Comment:
The paths shouldn't be hard-coded. The application should be configurable via command line. Also, cv::FrameSource can read "image directories" if you provide the filename as a "printf pattern". Please, see the documentation (http://docs.opencv.org/modules/highgui/doc/reading_and_writing_images_and_video.html#videocapture)